app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -295,6 +295,9 @@ class PipelineSampleReport extends React.Component {
     let selected_taxons = [];
     const thresholded_taxons = input_taxons || this.state.thresholded_taxons;
     const active_thresholds = this.state.activeThresholds;
+    const specificOnly =
+      this.state.readSpecificity.toLowerCase() === "specific only";
```

Comment:
I know it is not new, but how hard would it be to change it to 0/1?

Reply:
Seems reasonable. I will change it